app/src/main/java/com/nutomic/syncthingandroid/http/GetRequest.java

```diff
@@ -16,14 +16,14 @@
  */
 public class GetRequest extends ApiRequest {
 
-    public static final String URI_CONFIG      = "/rest/system/config";
-    public static final String URI_VERSION     = "/rest/system/version";
-    public static final String URI_SYSTEM      = "/rest/system/status";
-    public static final String URI_CONNECTIONS = "/rest/system/connections";
-    public static final String URI_MODEL       = "/rest/db/status";
-    public static final String URI_DEVICEID    = "/rest/svc/deviceid";
-    public static final String URI_REPORT      = "/rest/svc/report";
-    public static final String URI_EVENTS      = "/rest/events";
+    public static final String URI_CONFIG       = "/rest/system/config";
+    public static final String URI_VERSION      = "/rest/system/version";
+    public static final String URI_SYSTEM       = "/rest/system/status";
+    public static final String URI_CONNECTIONS  = "/rest/system/connections";
+    public static final String URI_STATUS       = "/rest/db/status";
+    public static final String URI_DEVICEID     = "/rest/svc/deviceid";
+    public static final String URI_REPORT       = "/rest/svc/report";
+    public static final String URI_EVENTS       = "/rest/events";
```

Comment:
Why the whitespace change?

Reply:
I did some tabbing due to URI_CONNECTIONS. Revert?

Reply:
I just wondered why you changed the other lines too, because there is no need to indent it any more. Now it looks fine.